### DISCUSSION TITLE
How to Customize the Color of the GridSplitter Handle?

### DISCUSSION THREAD

**Discussion by hamu777** [CommunityToolkit/Windows#553] [Q&A] [answered]
I would like to customize the color of the rectangle in the middle of the GridSplitter. Is there any way to achieve this?

```
        <controls:GridSplitter Grid.Column="1" Width="16" Background="Transparent" HorizontalAlignment="Left" VerticalAlignment="Stretch">
            <controls:GridSplitter.Resources>
                <x:Double x:Key="SizerBaseThumbHeight">100</x:Double>
                <x:Double x:Key="SizerBaseThumbWidth">1</x:Double>
                <x:Double x:Key="SizerBaseThumbRadius">2</x:Double>
                <Thickness x:Key="SizerBasePadding">4</Thickness>
            </controls:GridSplitter.Resources>
        </controls:GridSplitter>
```

**Comment by Arlodotexe** [comment]
@hamu777 It looks like this was closed unanswered. For those that search for this in the future, what approach did you end up going with?

**Comment by hamu777** [comment, marked as answer]
Resolved on my own. The Thumb color can be changed using the Foreground property. Thanks.
`<controls:GridSplitter  Foreground="Red" Grid.Column="0" Grid.Row="1"  Width="16" HorizontalAlignment="Right" VerticalAlignment="Stretch" IsThumbVisible="True"/>`
